src/Push/PushQueue.js

```diff
@@ -16,6 +18,7 @@ export class PushQueue {
   constructor(config: any = {}) {
     this.channel = config.channel || PushQueue.defaultPushChannel();
     this.batchSize = config.batchSize || DEFAULT_BATCH_SIZE;
```

Comment:
Do we still need batchSize?

Reply:
yeah we still need it so we have a query batch size and an enqueue batch size:

https://github.com/parse-community/parse-server/pull/4378/files#diff-6f234e6e995cfa72a42019190989aaa4R42